Backend/app/dashboard/rider-dashboard/src-read-only/API/Action/RiderPlatform/Management/Invoice.hs

```diff
@@ -26,7 +27,13 @@ type API = ("invoice" :> GetInvoiceInvoice)
 handler :: (Kernel.Types.Id.ShortId Domain.Types.Merchant.Merchant -> Kernel.Types.Beckn.Context.City -> Environment.FlowServer API)
 handler merchantId city = getInvoiceInvoice merchantId city
 
-type GetInvoiceInvoice = (ApiAuth 'APP_BACKEND 'CUSTOMERS 'INVOICE :> API.Types.RiderPlatform.Management.Invoice.GetInvoiceInvoice)
+type GetInvoiceInvoice =
+  ( ApiAuth
+      'APP_BACKEND_MANAGEMENT
```

Comment:
here `APP_BACKEND` changed to autogenerated `APP_BACKEND_MANAGEMENT`, because now `serverName`  is the same for all apis from current folder and defined in dhall.